src/java.desktop/share/native/libfontmanager/HBShaper_Panama.c

```diff
@@ -98,7 +98,6 @@ JDKEXPORT int jdk_hb_shape(
      float devScale = 1.0f;
      if (getenv("HB_NODEVTX") != NULL) {
          float xPtSize = euclidianDistance(matrix[0], matrix[1]);
-         float yPtSize = euclidianDistance(matrix[2], matrix[3]);
```

Comment:
I paused to think about whether the problem is really that  the division below should be using yPtSize .. in other words, the correct fix isn't always as obvious as it might look.

For the record, I think the unused var is becuase it was patterned after the code code in HBShaper.c which stores the yPtSize in a struct, but here it is clear that it isn't used.

    fi->xPtSize = euclidianDistance(fi->matrix[0], fi->matrix[1]);
    fi->yPtSize = euclidianDistance(fi->matrix[2], fi->matrix[3]);
    if (getenv("HB_NODEVTX") != NULL) {
        fi->devScale = fi->xPtSize / fi->ptSize;


Now, as far as I can they aren't used except in that fn, so probably can also go ..

Reply:
In addition, I've confirmed `euclidianDistance` does not make any side-effects.